identity/utils.py

```diff
@@ -0,0 +1,39 @@
+import time
+
+from jwcrypto import jwk, jwt
+
+
+SIGNING_ALG = "RS256"
+EXPIRY_TIME = 15 * 60  # 15 minutes
+
+
+def mint_access_jwt(key: jwk.JWK, urn: str) -> jwt.JWT:
+    """
+    Minting JWTs with the following claims:
```

Comment:
change to `Mint a JWT with the following claims:` and make similar changes to the rest of the docstring. (same with refresh)
the imperative mood is a bit easier to read

Reply:
good call